.flowconfig

```diff
@@ -30,9 +30,6 @@
 .*/integration-test/.*
 .*/sandbox/.*
 
-[libs]
```

Comment:
flow gets run from the top anyway, so might as well just keep the types at the root of the repo